# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/quota/moz.build
# Commit: a576f5346c6c
# Full Hash: a576f5346c6cfc32a5b147158e48ec1f4fcf2a8f
# Author: Jan Varga <jvarga@mozilla.com>
# Date: 2025-06-12 16:31:09
# Regressor Bug: 1962133
# File Overlap Count: 5
# Description:
#   Bug 1962133 - QM: Prepare metadata header infrastructure for accessed flag addition and reduce duplication; r=dom-storage-reviewers,hsingh
#   
#   This patch restructures metadata header reading and writing code to reduce
#   duplication and establish clean infrastructure for adding the accessed flag to
#   the metadata file header.
# ==============================================================================

diff -r f4284b0494fc -r a576f5346c6c dom/quota/moz.build
--- a/dom/quota/moz.build	Wed Jun 11 20:01:43 2025 +0000
+++ b/dom/quota/moz.build	Wed Jun 11 20:09:11 2025 +0000
@@ -124,6 +124,7 @@
     "ClientUsageArray.cpp",
     "DecryptingInputStream.cpp",
     "DirectoryLockImpl.cpp",
+    "DirectoryMetadata.cpp",
     "EncryptingOutputStream.cpp",
     "FileStreams.cpp",
     "FileUtils.cpp",
